lib/msal-browser/src/broker/EmbeddedClientApplication.ts

```diff
@@ -149,6 +140,7 @@ export class EmbeddedClientApplication {
             messageChannel.port1.onmessage = ((message: MessageEvent): void => {
                 this.logger.verbose(`in messageBroker<T> w/ origin: ${message}`);
                 clearTimeout(timeoutId);
+                messageChannel.port1.close();
```

Comment:
port1 and port2, if they have significance, can we please add comments?

Reply:
This is part of the MessageChannel API and is documented on MDN. I did also add a brief comment to explain what this is.